docs/fastdds/property_policies/statistics.rst

```diff
@@ -0,0 +1,10 @@
+.. include:: ../../03-exports/aliases.include
+.. include:: ../../03-exports/aliases-api.include
+
+.. _property_policies_statistics:
+
+Statistics Module Settings
```

Comment:
I'd say this is a non-consolidated QoS that we hope to promote to extension one day